res/controllers/Denon-MC7000-scripts.js

```diff
@@ -890,17 +1037,50 @@ MC7000.censor = function(channel, control, value, status, group) {
     }
 };
 
+// Param Button for Pitch Play to increase or decrease pitch, Star rating otherwise
+MC7000.StarsDown = function(channel, control, value, status, group) {
+    var deckNumber = script.deckFromGroup(group);
+    var deckIndex = deckNumber - 1;
+    if (value === 0x00) {
+        //return; // don't respond to note off messages
+    } else {
+        if (MC7000.PADModePitch[deckIndex]) {
+            for (var i = 0; i < 8; i++) {
+                MC7000.halftoneToPadMap[deckIndex][i] = MC7000.halftoneToPadMap[deckIndex][i] - 8; // pitch down
+            }
+        } else {
+            engine.setValue(group, "stars_down", true); // stars down
+        }
+    }
+};
+
+MC7000.StarsUp = function(channel, control, value, status, group) {
```

Comment:
```suggestion
// Param Button for Pitch Play to increase pitch, or increase star rating otherwise
MC7000.StarsUp = function(channel, control, value, status, group) {
```

Reply:
done